core/domain/suggestion_services.py

```diff
@@ -1108,6 +1142,37 @@ def get_community_contribution_stats():
         community_contribution_stats_model)
 
 
+def get_suggestion_types_that_need_reviewers():
+    """Uses the community contribution stats to determine which suggestion
+    types need more reviewers.
+
+    Returns:
+        dict. A dictionary where the keys are suggestion types and each value
+        corresponds to a set that contains the language codes within the
+        suggestion type that need more reviewers. For example, for translation
+        suggestions, the value would be a set of language codes that
+        translations are offered in that need more reviewers.
+    """
+    suggestion_types_need_more_reviewers = {}
+    stats = get_community_contribution_stats()
+
+    language_codes_that_need_reviewers = (
+        stats.get_translation_language_codes_that_need_reviewers()
+    )
+    if len(language_codes_that_need_reviewers) != 0:
+        suggestion_types_need_more_reviewers[
+            suggestion_models.SUGGESTION_TYPE_TRANSLATE_CONTENT] = (
+                language_codes_that_need_reviewers
+            )
+
+    if stats.are_question_reviewers_needed():
+        suggestion_types_need_more_reviewers[
+            suggestion_models.SUGGESTION_TYPE_ADD_QUESTION] = {
```

Comment:
It's a bit odd to me that we're binding a language code the question type since it's not actually relevant to question suggestions. Could we have the value of the dict be dependent on the type? E.g. for question could we just bind an empty dict since it's the presence that matters?

Reply:
It's now an empty dict